frame/evm/precompile/bn128/Cargo.toml

```diff
@@ -13,7 +13,7 @@ sp-core = { version = "3.0.0", default-features = false, git = "https://github.c
 sp-io = { version = "3.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
 fp-evm = { version = "1.0.0", default-features = false, path = "../../../../primitives/evm" }
 evm = { version = "0.25.0", default-features = false, features = ["with-codec"] }
-bn = { package = "substrate-bn", version = "0.5", default-features = false }
+bn = { package = "substrate-bn", version = "0.6", default-features = false }
```

Comment:
Please bump `pallet-evm-precompile-bn128` to `2.0.0-dev` since I think this is not backward compatible!